src/components/Accordion.vue

```diff
@@ -17,11 +17,18 @@
   cursor: pointer;
   display: flex;
   align-items: center;
-  color: #88170e;
+  color: #800080;
+  margin-top:10px;
+  margin-bottom:10px;
+  margin-left: 110px;          /* This VERY POOR SUBSTITUTE for real centering */
```

Comment:
since this is a flex container what you want is "justify-content: center"

Reply:
p.s. i fixed this here: https://github.com/edwardsharp/ecotopia-today/commit/9a6f0cfca1893dce51683525dc602eafec3ab4da